### ISSUE TITLE
Improve logrotate script

### ISSUE THREAD

**Issue by TJC** [louiszuckerman/glusterfs-debian#1]
I notice that this Debian package installs a very minimal, and non-functional, logrotate script.
There appears to be a better one here:
https://github.com/gluster/glusterfs/blob/master/extras/glusterfs-logrotate

**Comment by louiszuckerman** [comment]
Sorry for the extreme delay responding to this issue.  This repository spent a lot of time dormant and I just now noticed your issue.  Others have brought the logrotate situation to my attention recently and I am willing to update the config file in the next release.

Since this problem doesn't affect me directly I'd like to get more input from the community.  Does this issue affect you?  Does the linked logrotate config file solve your problems?  Does it need further improvement?

Any & all feedback is welcome.

Thank you!

**Comment by TJC** [comment]
Sorry, we abandoned glusterfs ages ago. I'll unsub from this bug.

**Comment by partneroid-zz** [comment]
Yes, this needs to be fixed, the logrotation is mostly broken and a working version has been committed in 2011. Every single Debian GlusterFS user needs to fix themselves both server and client side log rotations which simply makes no sense since there is a functional one available. This is a must-have in the packages.

**Comment by louiszuckerman** [comment]
To be clear, the proposal is to replace the existing logrotate configs in glusterfs-server & glusterfs-client with the configuration provided in the extras/ directory of the glusterfs source, available here: https://github.com/gluster/glusterfs/blob/master/extras/glusterfs-logrotate

I am willing to make this change, and intend to do so in the next release, based on feedback from @partneroid.  If anyone has any objections or further improvements please voice them in comments here.

Thank you.
